.github/workflows/test.yml

```diff
@@ -7,7 +7,8 @@ jobs:
     runs-on: ubuntu-latest
     name: Branch Name
     steps:
-      - uses: AlbertHernandez/branch-name-action@v1
+      - uses: actions/checkout@v4
```

Comment:
**issue (blocking):** I see this still failing (https://github.com/AlbertHernandez/branch-name-action/actions/runs/7882599800/job/21532232107), not sure if we should use the ref of the current context or other thing, let's investigate how is the best way to point to our current action, if not, we can always have this in the previous way and improve it in the next PR

Reply:
This error appears because you are trying to launch an action whose tag does not exist in the main branch. With the change I provide, the test workflow will trigger the local action. However, it cannot be tested until the change reaches the main branch.

Reply:
I think we can solve this issue pointing to the current branch, this checkout action also accept a property named ref, if this ref is the branch name we are working, most probably this will work